core/src/cli.rs

```diff
@@ -181,15 +177,6 @@ where
 {
     let args = parse_from(itr).wrap_err("Failed to parse CLI arguments.")?;
 
-    let level_filter = match args.verbose {
-        0 => None,
```

Comment:
moved to `main.rs` since initializing logger in a `get` function doesn't make sense